one_electron.py

```diff
@@ -0,0 +1,151 @@
+from orbital4c import complex_fcn as cf
+from orbital4c import orbital as orb
+from orbital4c import operators as oper
+from scipy.constants import hbar
+from scipy.linalg import eig, inv
+from scipy.special import legendre, laguerre, erf, gamma
+from vampyr import vampyr3d as vp
+import numpy as np
+import numpy.linalg as LA
+import sys, getopt
+
+def gs_D_1e(spinorb1, potential, mra, prec, der):
+    print('Hartree-Fock 1e')
+    
+    error_norm = 1
+    compute_last_energy = False
+    
+    P = vp.PoissonOperator(mra, prec)
+    light_speed = spinorb1.light_speed
+
+    while (error_norm > prec or compute_last_energy):
+        n_22 = spinorb1.overlap_density(spinorb1, prec)
+
+        # Definition of two electron operators
+        B22    = P(n_22.real) * (4 * np.pi)
```

Comment:
The names are still reminescent of the 2e code. There is no 2nd electron here :-)

Reply:
Changed into n_11 and B11